### PR TITLE
#169723099 Set up the backend

### PR DESCRIPTION
**Pull request by kcharles52**
### What PR does?
- Sets up the backbone for myarcade project
### How to test the PR?
- run the following commands
- `git fetch ch-backend-project-setup-169723099`
- `git checkout ch-backend-project-setup-169723099`
- `npm run dev`
- visit `localhost:4000`
- You receive a JSON object `{"message":"welcome to myarcade"}`
### Stories associated with PR

[#169723099](https://www.pivotaltracker.com/story/show/169723099)

### Screenshots if any
- N/A

### CODE REVIEW

**Review by missvicki — APPROVED** [review]
LGTM

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    }]
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-object-rest-spread",
+    ["module-resolver", {
+      "root": ["./api/**","./api/config**","./api/graphql**"]
+    }]
+  ]
+}
```
Comment (katunold) [inline .babelrc]: Kindly add an empty line at the end of this file

File: nodemon.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "watch": [
+    "api/**/*.js",
+    "api/**/*.graphql",
+    "api/**/*.gql"
+  ],
+  "env": {
+    "NODE_ENV": "development"
+  },
+  "ext": "js,json,graphql"
+}
```
Comment (katunold) [inline nodemon.json]: Newline character here too

File: package.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "myarcade-backend",
+  "version": "1.0.0",
+  "description": "This is an API for a tenant management system for a shopping mall or arcade",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel api --out-dir dist",
```
Comment (katunold) [inline package.json:7]: I don't see a test script, are we not going to write tests?

**Review by katunold — APPROVED** [review]
I have left some little feedback for you otherwise this looks good

File: package.json (hunk 3 above)
Comment (kcharles52) [reply package.json:7]: I saw a story for it. So the setup for jest will be done later

**Comment by katunold** [comment]
Well done @kcharles52, LGTM

**Pull request merged**